microcosm/tests/test_loaders.py

```diff
@@ -50,6 +51,32 @@ def configfile(data):
     yield configfile_
 
 
+def test_expand_config():
+    """
+    Configuration expansion should work.
+
+    """
+    assert_that(
+        expand_config(
+            {
+                "prefix.foo": "bar",
+                "prefix.BAR.baz": lambda: "foo",
```

Comment:
Where does the desire for the default behavior to call values of the dictionary if they are callable come from?

It seems undesirable, for example, if they are handler mappings from microcosm pubsub and you only want to call them when you have a message to process. I like having the ability to special a value func, but not sure why calling is the default behavior.